EnglishFilter/sections/antiadblock.txt

```diff
@@ -781,6 +781,8 @@ btcbunch.com#%#//scriptlet("abort-on-property-write", "TestAd")
 @@||googletagmanager.com/gtag/js$domain=btcbunch.com
 @@||static.surfe.pro/js/net.js$domain=btcbunch.com
 !#endif
+! https://github.com/AdguardTeam/AdguardFilters/issues/115641
```

Comment:
We already have the rules for this script, just add new domain to the necessary scriptlet and URL rules
https://github.com/AdguardTeam/AdguardFilters/blob/58fd6094784a6d77e66e3ac309b496d3c8bc1ad2/EnglishFilter/sections/antiadblock.txt#L567

Comment has the word which helps to find the rules easily
![image](https://user-images.githubusercontent.com/8361299/163567725-8219a661-d816-44af-aa94-cc256421ffd2.png)

Reply:
Yep, but it's not necessary here.